src/app/Console/Commands/Install.php

```diff
@@ -26,45 +29,182 @@ class Install extends Command
+        $this->createUsers();
+
+        // Addons
+        $this->installAddons();
+
+        // Done
+        $url = Str::of(config('app.url'))->finish('/')->append('admin/');
+        $this->infoBlock('Backpack installation complete.', 'done');
+        $this->note("Go to <fg=blue>$url</> to access your new admin panel.");
+        $this->note('You may need to run `php artisan serve` to serve your Laravel project.');
+        $this->newLine();
+    }
+
+    private function createUsers()
+    {
+        $userClass = config('backpack.base.user_model_fqn', 'App\Models\User');
+        $userModel = new $userClass();
+
+        // Count current users
+        $currentUsers = $userModel->count();
+
+        $this->newLine();
+        $this->infoBlock('Creating an admin:');
+        $this->note('Quickly jump in your admin panel, using the email & password you choose here.');
+        $this->note('Currently there '.trans_choice("{0} are <fg=blue>no users</>|{1} is <fg=blue>1 user</>|[2,*] are <fg=blue>$currentUsers users</>", $currentUsers).' in the database.');
+
+        $total = 0;
+        while ($this->confirm(' Add '.($total ? 'another' : 'an').' admin user?')) {
+            $name = $this->ask(' Name');
+            $mail = $this->ask(" {$name}'s email");
+            $pass = $this->secret(" {$name}'s password");
+
+            try {
+                $userModel->insert([
+                    'name' => $name,
+                    'email' => $mail,
+                    'password' => bcrypt($pass),
+                    'created_at' => Carbon::now(),
+                    'updated_at' => Carbon::now(),
+                ]);
+
+                $this->deleteLines(12);
+                $this->progressBlock('Adding admin user');
+                $this->closeProgressBlock();
+                $this->note($name);
+                $this->note($mail);
+
+                $total++;
+            } catch (\Throwable$e) {
+                $this->errorBlock($e->getMessage());
+            }
+        }
+
+        $this->deleteLines(1);
+    }
+
+    private function isEveryAddonInstalled()
+    {
+        return collect($this->addons)->every(function ($addon) {
+            return file_exists($addon->path);
+        });
+    }
+
+    private function updateAddonsStatus()
+    {
+        $this->addons = $this->addons->each(function (&$addon) {
+            $isInstalled = file_exists($addon->path);
+            $addon->status = $isInstalled ? 'installed' : 'not installed';
+            $addon->statusColor = $isInstalled ? 'green' : 'yellow';
+        });
+    }
+
+    private function installAddons()
+    {
+        // map the addons status
+
+        $this->addons = collect($this->addons)
+            ->map(function ($class) {
+                return (object) $class::$addon;
+            });
+
+        $this->updateAddonsStatus();
+
+        // if all addons are installed do nothing
+        if ($this->isEveryAddonInstalled()) {
+            return;
+        }
```

Comment:
Haha I just noticed this when running `php artisan backpack:install` on a project with everything already installed. Nice touch! 👏👏👏